pygaggle/run/evaluate_passage_ranker.py

```diff
@@ -137,7 +149,8 @@ def main():
                          seq_class_transformer=construct_seq_class_transformer,
                          random=lambda _: RandomReranker())
     reranker = construct_map[options.method](options)
-    evaluator = RerankerEvaluator(reranker, options.metrics)
+    writer = MsMarcoWriter(args.output_file, args.overwrite_output)
```

Comment:
Should we add an argument that allows the user to change between standard TREC and MS MARCO formats?

Reply:
I will add this when I add TREC support. Right now it is non-existent anyways!